lib/ae_page_objects/elements/collection.rb

```diff
@@ -56,6 +56,13 @@ def configure(options)
       @item_locator = options.delete(:item_locator) || default_item_locator
     end
 
+    def options
+      @options ||= begin
+        evaled_locator = eval_locator(@item_locator).clone
```

Comment:
For consistency, I think this, and other instances, should use `dup` rather than `clone`. (There is also some extra overhead with `clone` because it does more than `dup`, and in this case the extra things it does aren't necessary.)